### ISSUE TITLE
Can't show the main window correctly.

### ISSUE THREAD

**Issue by zzan1** [nomacs/nomacs#508]
**Describe the bug**
Can't show the window in windows 10. I think it might relate with the double screen set.

**To Reproduce**
1. Open the executor or open a picture.
2. The soft started normally, and I can see the icon in the tasks view. 
3. But there no window showed correctly in the desktop.

**Expected behavior**
Show the main window.

**Screenshots**
![present](https://user-images.githubusercontent.com/47623989/85382703-cfc5cc80-b571-11ea-9240-324a2d0974b3.gif)

**Desktop (please complete the following information):**
 - OS: windows 10 family version
 - Version:18363.9000

**Additional context**

**Comment by diemmarkus** [comment]
> I think it might relate with the double screen set.

yes. This can happen if you change from a double screen to single screen setup (nomacs is actually shown outside the visible viewport). You can fix this locally:
- Open a Windows Explorer and type `%appdata%\nomacs\Image Lounge`
- Open `settings.ini` with a text editor
- Delete the line `geometryNomacs=@Rect(1295 453 850 504)`

I am leaving this open, as it would be nice if nomacs defaulted on its own...

**Comment by zzan1** [comment]
Yes， it works. Thanks you very much!!!🙏

**Comment by leejuyuu** [comment]
@zzan1, I think this has been fixed. Could you try the latest version?

**Comment by leejuyuu** [comment]
Closing as this is fixed.
